config/Caddyfile

```diff
@@ -1,3 +1,3 @@
-{$MW_SITE_FQDN}:{$HTTPS_PORT}
+{$MW_SITE_FQDN}, http://localhost
```

Comment:
This seems to be unnecessary, `MW_SITE_FQDN` is configured via `.env` and if you'd like to add option to run the stack without TLS on localhost it's better to introduce some extra setting variable rather than always make it listen for localhost